stable/nginx-ingress/values.yaml

```diff
@@ -64,9 +64,25 @@ controller:
   ##   default-ssl-certificate: "<namespace>/<secret_name>"
   extraArgs: {}
 
+  ## Indicates whether the "extraEnvs" value should be inlined to the deployment/daemonset in its raw form,
+  ## or injected as literal key-value pairs.
+  ##
+  ## NOTE: this defaults to false for backwards-compatability reasons. This is subject
+  ## for deprecation in the future, so it is encouraged to set this to true.
+  inlineExtraEnvs: true
```

Comment:
should be false.

Reply:
😱 
442fd0c46aba95ea90e2ca1e30b77c3475916d2e